Code/Framework/AzFramework/AzFramework/Quality/QualityCVarGroup.h

```diff
@@ -57,15 +58,18 @@ namespace AzFramework
     private:
         AZ_DISABLE_COPY(QualityCVarGroup);
 
-        ValueType GetHighestQualityForSetting(AZStd::string_view path);
-        void PerformConsoleCommand(AZStd::string_view command, AZStd::string_view key);
+        AZ::PerformCommandResult PerformConsoleCommand(AZStd::string_view command, AZStd::string_view key);
 
-        AZ::IConsole* m_console;
-        using QualityCVarGroupFunctor = AZ::ConsoleFunctor<QualityCVarGroup, /*replicate=*/true>;
+        inline static constexpr bool ReplicateCommand = true;
+        using QualityCVarGroupFunctor = AZ::ConsoleFunctor<QualityCVarGroup, ReplicateCommand>;
         AZStd::unique_ptr<QualityCVarGroupFunctor> m_functor;
-        AZ::u16 m_numQualityLevels;
+        AZStd::string m_name;
         AZStd::string m_path;
-        ValueType m_qualityLevel;
-        AZ::SettingsRegistryInterface* m_settingsRegistry;
+        ValueType m_qualityLevel = QualityLevels::DefaultQualityLevel;
     };
 } // namespace AzFramework
+
+namespace AZ
+{
+    AZ_TYPE_INFO_SPECIALIZE(AzFramework::QualityLevels, "{9AABD1B2-D433-49FE-A89D-2BEF09A252C0}");
+}

```

Comment:
This doesn't have to be in the AZ namespace.
This was changed so that TypeInfo can be specialized in the namespace of the class/enum/struct/union declaration in this [PR](https://github.com/o3de/o3de/pull/14824).

One of the problems with specialization using the AZ_TYPE_INFO_SPECIALIZE macro outside of the types namespace, is that the TypeName associated is the fully qualified name of "AzFramework::QualityLevels"
That name would also get serialized to output if using the SerializeContext.
Also for the BehaviorContext and scripting usage the name would be reflected as `AzFramework__QualityLevels` in lua

I recommend the reflection be moved right after the declaration of the enum class above without the class fully qualified name
`AZ_TYPE_INFO_SPECIALIZE(QualityLevels, "{9AABD1B2-D433-49FE-A89D-2BEF09A252C0}");`

If a custom name is desired for reflection to something such as the SerializeContext/BehaviorContext, the `AZ_TYPE_INFO_SPECIALIZE_WITH_NAME` macro can be used to specify a reflection name that is different from the C++ type name and is not limited to the C++ identifier grammar.

Reply:
Ah I read the comments wrong in `TypeInfoSimple.h`  I thought they said it has to be declared in "namespace AZ", even though it says `NOT need has to be`.